packages/target-nodejs-sdk/src/index.js

```diff
@@ -231,12 +241,23 @@ export default function bootstrap(fetchApi) {
         },
         logger: this.logger,
         useBeacon: true,
-        ...options
+        ...request
```

Comment:
Let's revert this back to options, since it's basically an envelope around the request + other metadata.  If we use the spread operator on request here then all the request fields will get assigned to targetOptions - which we don't want.